products/fairphone.md

```diff
@@ -10,6 +10,14 @@ releaseColumn: false
 releaseDateColumn: true
 
 releases:
+
+-   releaseCycle: "5"
+    highestAndroidVersion: "12"
+    releaseDate: 2023-09-14
```

Comment:
It will be launched in two weeks, will keep this PR as draft in the mean time.

Reply:
Yup, I just wanted to make it ready :pray: